packages/design-system/src/components/popup/index.js

```diff
@@ -103,12 +104,21 @@ function Popup({
       }
       setPopupState({
         offset: anchor.current
-          ? getOffset(placement, spacing, anchor, dock, popup, isRTL, topOffset)
+          ? getOffset({
```

Comment:
this is just way too many props to pass them individually, just making it an object instead.